RuneWordFinder4/wwwroot/Source/app.js

```diff
@@ -1,20 +1,14 @@
-﻿import ES6Lib from './es6codelib';
-import 'bootstrap/dist/css/bootstrap.min.css';
+﻿import 'bootstrap/dist/css/bootstrap.min.css';
 import '../css/site.css';
 
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Counter from './reactcomponent';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import RuneWordFinder from './RuneWordFinder';
 
 ReactDOM.render(
-    <Counter />,
```

Comment:
The Counter class is a legacy of the tutorial and can be removed entirely.

Reply:
On second thought, the checkboxes will need to be replaced by a counter of sorts. While the current `Counter` may not fit that bill, it may be a nice start point.